### DISCUSSION TITLE
How to set the minimum width of the plugin？

### DISCUSSION THREAD

**Discussion by qingtingzhiyu39** [eclipse-theia/theia#14682] [General]
Hi,
  I have a question that how to set min-width for my widget.
  this is my widget,because my widget's width is very wide。Therefore, it is not possible to view all the header data during initialization。

 so, How to set the minimum width of the plugin？

<img width="1152" alt="无标题" src="https://github.com/user-attachments/assets/fe97b146-2bca-46c4-af6e-9fde5341f957" />

**Comment by msujew** [comment]
Hey @qingtingzhiyu39,

I don't think there is a way to limit the size of a widget. I would recommend to make your custom widget responsive instead, so that it works under any viewport.
